dev/com.ibm.ws.microprofile.faulttolerance.2.1.cdi/test/src/com/ibm/ws/microprofile/faulttolerance21/cdi/config/impl/CircuitBreakerConfig21Test.java

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.faulttolerance21.cdi.config.impl;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.lang.reflect.Method;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+import org.eclipse.microprofile.config.spi.ConfigProviderResolver;
+import org.eclipse.microprofile.faulttolerance.CircuitBreaker;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.ibm.ws.microprofile.config13.impl.Config13ProviderResolverImpl;
+import com.ibm.ws.microprofile.faulttolerance.spi.CircuitBreakerPolicy;
+import com.ibm.ws.microprofile.faulttolerance.spi.FaultToleranceProviderResolver;
+import com.ibm.ws.microprofile.faulttolerance20.impl.ProviderResolverImpl20;
+
+@SuppressWarnings("restriction")
+public class CircuitBreakerConfig21Test {
+
+    @Before
+    public void before() {
+        ConfigProviderResolver.setInstance(new Config13ProviderResolverImpl());
+        FaultToleranceProviderResolver.setInstance(new ProviderResolverImpl20());
+    }
+
+    @After
+    public void after() {
+        ((Config13ProviderResolverImpl) ConfigProviderResolver.instance()).shutdown();
+        ConfigProviderResolver.setInstance(null);
+        FaultToleranceProviderResolver.setInstance(null);
+    }
+
+    @Test
+    public void defaultCircuitBreakerPolicyTest() throws NoSuchMethodException, SecurityException {
```

Comment:
Not sure about this test name.

It reads like you're testing the `CircuitBreakerPolicy`, but this is a test of the `CircuitBreakerConfig`

Also, you'd usually have `test` first.

I might change this to `testDefaultValues`, or `testCbConfigDefaultValues`

Similarly for `customCircuitBreakerPolicyTest`.